src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -791,6 +792,24 @@ public void testPlaceInstanceHoldRequestTimeout(TestContext context) throws Exce
       .body("errorMessage", is(MSG_REQUEST_TIMEOUT));
   }
 
+  @Test
+  public void testPlaceInstanceHoldNoBody(TestContext context) throws Exception {
+    logger.info("=== Test place instance hold request with no request body ===");
+
+    mockOkapi.setDelay(requestTimeoutMs * 3);
+    RestAssured
+      .with()
+      .contentType(APPLICATION_JSON)
+      .post(
+          String.format("/patron/account/%s/instance/%s/hold?apikey=%s", patronId, instanceId,
+              apiKey))
+      .then()
+      .contentType(APPLICATION_JSON)
+      .statusCode(400)
+      .body("code", is(400))
+      .body("errorMessage", is(MSG_HOLD_NOBODY));
```

Comment:
Personally, I prefer not to reuse production constants in tests because it mixes the expectation and the actual. Meaning that if the constant is wrong, the tests won't reveal that.

Reply:
It was, but I hadn't considered your point.  I *could* change all the tests to not use production constants-there are not that many of them in this module.  If we don't do that then I'd rather leave as-is, as the inconsistency could cause confusion.  I know I'd be asking "why does every other test use constants, but this one does not?" if I didn't know.

Reply:
> I _could_ change all the tests to not use production constants-there are not that many of them in this module. If we don't do that then I'd rather leave as-is, as the inconsistency could cause confusion. I know I'd be asking "why does every other test use constants, but this one does not?" if I didn't know.

I agree that the in-consistency would be confusing (although that tends to the outcome of not changing anything unless it can be done atomically).

If the change is small, I suggest doing it now. This is entirely optional, and this pull request can be merged either way IMO.